core/storage/topic/gae_models_test.py

```diff
@@ -0,0 +1,147 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test for Topic model."""
+
+from constants import constants
+from core.domain import subtopic_page_domain
+from core.domain import subtopic_page_services
+from core.domain import topic_domain
+from core.domain import topic_services
+from core.platform import models
+from core.tests import test_utils
+import feconf
+
+(topic_models,) = models.Registry.import_models([models.NAMES.topic])
+
+
+class TopicModelUnitTest(test_utils.GenericTestBase):
+    """Test the TopicModel class."""
+
+    def setUp(self):
+        super(TopicModelUnitTest, self).setUp()
+        self.topic = topic_domain.Topic.create_default_topic(
+            topic_id='topic_id',
+            name='topic_test'
+        )
+        topic_services.save_new_topic(feconf.SYSTEM_COMMITTER_ID, self.topic)
+
+    def test__trusted_commit(self):
+        # Compares topic model class variable (version)
+        # assigned in _trusted_commit method.
+        self.assertEqual(self.topic.version, 1)
+
+    def test_get_by_name(self):
+        self.assertEqual(
+            topic_models.TopicModel.get_by_name(self.topic.name).name,
+            self.topic.name
+        )
+        self.assertEqual(
+            topic_models.TopicModel.get_by_name(self.topic.name).id,
+            self.topic.id
+        )
+
+
+class TopicCommitLogEntryModelUnitTest(test_utils.GenericTestBase):
+    """Test the TopicCommitLogEntryModel class."""
+
+    def setUp(self):
+        super(TopicCommitLogEntryModelUnitTest, self).setUp()
+        # Calling create() method calls _get_instance (a protected method)
```

Comment:
Same thing, this does not need to be in setUp().

Reply:
This was also done in commit: https://github.com/oppia/oppia/pull/5689/commits/74f5324cafc14d937b184f68081190bb6c943250